### PR TITLE
Mandrill

### PR DESCRIPTION
**Pull request by Fernan2**
Change Sendgrid to Mandrill. Also, now is the engine, and not the main app, who effectively sends to our mail provider

### CODE REVIEW

File: app/controllers/news_tematica/news_tematicas_controller.rb [hunk 1]
```diff
@@ -105,12 +105,7 @@ def carga_variables_preview(news_tematica)
     def dame_html
       html_limpio = render_to_string('news_tematica/news_tematicas/_preview', layout: false)
       html_inlineado = Premailer.new(html_limpio, with_html_string: true, input_encoding: 'UTF-8').to_inline_css
-      restaura_tags_sendgrid html_inlineado
-    end
-
-    def restaura_tags_sendgrid(html)
-      html.gsub('%5B', '[')
-          .gsub('%5D', ']')
+      html_inlineado.gsub("%7C", "|")
```
Comment (Fernan2) [inline app/controllers/news_tematica/news_tematicas_controller.rb]: Not sure about both being valid for special characters like those, I'll keep as is

**Comment by scruti** [comment]
:+1:

**Comment by d2bit** [comment]
:+1:

**Comment by 007lva** [comment]
:+1:

**Pull request merged**